test/evergreen/print_stack_trace.py

```diff
@@ -130,24 +130,24 @@ def main():
 
         # Get the executable from the core file itself.
         proc = subprocess.Popen(["file", core_file_path], stdout=subprocess.PIPE)
-        executable_path = str(proc.communicate())
-        start = executable_path.find('execfn: ')
-        end = executable_path.find('platform:')
-        executable_path = executable_path[(start+9):(end-3)]
+        # The file command prints something similar to:
+        # WT_TEST/test_practice.0/dump_python3.16562.core: ELF 64-bit LSB core file x86-64, version 1 (SYSV),
+        # SVR4-style, from 'python3 /home/ubuntu/wiredtiger/test/suite/run.py -j 1 -p test_practice',
+        # real uid: 1000, effective uid: 1000, real gid: 1000, effective gid: 1000, execfn: '/usr/bin/python3', platform: 'x86_64'
+        output = str(proc.communicate())
 
-        # Extract the filename from the core file path, to create a stacktrace output file.
-        file_name, _ = os.path.splitext(os.path.basename(core_file_path))
-        file_name = file_name + ".stacktrace.txt"
```

Comment:
@jiechenbo, was there a specific reason we wanted to upload the core dump as a text file in Evergreen?

Reply:
the idea was to print the stacktrace of the coredump because the coredump can be potentially quite big and you would need the original binary. Therefore i chose to place it into a text file instead.

Reply:
Thanks, @jiechenbo. I think it is fine to print everything in the logs, even if there are many threads. We can always revisit this if it's actually an issue.

Reply:
Oh I see, I still actually prefer that we upload the stack traces of each coredump as a file here. The reason is because I saw that there was too much output and was an issue in the logs if we output all the threads in the logs. Another reason is that users will need to go into the logs scroll all the way down to see the specific failure they wanted. Although this is just one extra click, im inclined that we keep this feature. @etienneptl is there an upside for removing this?

Reply:
We simply thought it would be fine to dump everything in the logs but I get the struggle of needing to scroll all the way down until you find the stack trace you want, especially if there are multiple core dumps.
@Siddhartha8899, are you ok with leaving the code as is and uploading the stack trace to a file?

Reply:
Hmm, I do not mind either way tbh. However, just to be clear, we wish to not `dump_all` in the logs and `dump_all` in the file. Is that correct?

@etienneptl scrolling all the way down seems easier than downloading the file and looking at the contents, no?

@jiechenbo do you have an example patch or test of logs for me to better visualize this problem? The patch that I have in my ticket (where I deliberately crash, looks good to me).

Reply:
> @etienneptl scrolling all the way down seems easier than downloading the file and looking at the contents, no?

I believe so, but the experience might not be the same for everyone.

Reply:
[This](https://evergreen.mongodb.com/task_log_raw/wiredtiger_ubuntu2004_long_test_patch_c522dcecaee4091682cdbba66353df47a9f14e27_6358b3ba0ae60669d7c0e408_22_10_26_04_12_57/0?type=T) could be an example of the problem that @jiechenbo might be referring too. Does it look that bad? Looks okay to me.

Reply:
@Siddhartha8899  I believe that is a good example. There seems to be multiple test failures in that ticket, and it would be hard to filter through all the logs of each coredump. Currently the usage is that there is a stacktrace for each coredump you have. And also you don't need to download the file if its an txt file as it will just open it on a new tab for you.

Reply:
Updated in [79daf14](https://github.com/wiredtiger/wiredtiger/pull/8377/commits/79daf14026c5f96cac8085e03942eb22a1e8d86d)